labs/github/github.py

```diff
@@ -0,0 +1,79 @@
+import requests
+from labs.config import GITHUB_ACCESS_TOKEN
+from labs.config import GITHUB_API_BASE_URL
+from labs.config import GITHUB_OWNER
+from labs.config import GITHUB_REPO
+from labs.config import GITHUB_USERNAME
+from labs.config import get_logger
+
+class Github:
```

Comment:
nit: recommend changing the name to GithubRequests or something else that lets us know we are interacting with it, but not on the tradicional commit way

Reply:
a dataclass would simplify this class definition https://docs.python.org/3/library/dataclasses.html